charts/opentelemetry-demo/values.yaml

```diff
@@ -385,6 +389,27 @@ components:
       runAsGroup: 101
       runAsNonRoot: true
 
+  imageprovider:
+    enabled: true
+    useDefault:
+      env: true
+    service:
+      port: 8081
+    env:
+      - name: IMAGE_PROVIDER_PORT
+        value: "8081"
+      - name: OTEL_COLLECTOR_PORT_GRPC
+        value: "4317"
+      - name: OTEL_COLLECTOR_HOST
+        value: $(OTEL_COLLECTOR_NAME)
+    resources:
+      limits:
+        memory: 50Mi
+    securityContext:
+      runAsUser: 101  # nginx
+      runAsGroup: 101
+      runAsNonRoot: true
+
```

Comment:
is this working for you?  it is triggering `/bin/sh: 1: cannot create /etc/nginx/nginx.conf: Permission denied` here